test/txsim/blob.go

```diff
@@ -55,6 +56,11 @@ func (s *BlobSequence) WithShareVersion(version uint8) *BlobSequence {
 	return s
 }
 
+func (s *BlobSequence) WithGasPrice(gasPrice float64) *BlobSequence {
+	s.gasPrice = gasPrice
+	return s
+}
+
```

Comment:
used this in tests to set a lower gas price than other normal txs, maybe not needed